backend/meeting/src/api/meeting.js

```diff
@@ -1,5 +1,9 @@
 const MeetingService = require("../services/meeting-service");
-const { SubscribeMessage, ValidateMeetingInput, PrintFormattedMessage } = require("../utils");
+const {
+	SubscribeMessage,
+	ValidateMeetingInput,
+	PrintFormattedMessage,
+} = require("../utils");
```

Comment:
This is due to my prettier line break is set to 100 while yours is set to 80 or something